app/controllers/admin/tutorials_controller.rb

```diff
@@ -5,12 +5,14 @@ def edit
 
   def create
     tutorial = Tutorial.new(tutorial_params)
-    if tutorial.save
-      flash[:success] = 'Successfully created tutorial.'
-      redirect_to tutorial_path(id: tutorial.id)
+    if tutorial.save && playlist_id != ''
```

Comment:
Hard to believe how crazy this thing used to look and how reasonable it looks now!